lib/credentials/ecs_credentials.d.ts

```diff
@@ -1,17 +0,0 @@
-import {Credentials} from '../credentials';
```

Comment:
Can you restore this file? This change leaves the `ECSCredentials` class in place and preserves its interface, so we would still want to provide types for that class.